docs/TEAMS.md

```diff
@@ -0,0 +1,202 @@
+The Organic Maps is led by contributors from across the community. Different functional domains may periodically establish, develop, or dissolve their own leadership structures, whether formal or informal.
+
+The file contains the functional project structure. The structure of GitHub teams must match the content of this file. Any changes to memberships must be performed via a PR to this file.
```

Comment:
If the sole purpose of this doc is to match to the Github teams which match to the CODEOWNERS file then team members should be limited to reviewers only and then indeed there is no need to duplicate members in this file - having them in Github teams is enough - and people would apply for membership via creating issues.

Reply:
Let's rely on GitHub Teams.